packages/neo-one-client/src/__tests__/DeveloperClient.test.js

```diff
@@ -0,0 +1,21 @@
+/* @flow */
+import DeveloperClient from '../DeveloperClient';
+import NEOONEDataProvider from '../provider/neoone/NEOONEDataProvider';
+
+describe('DeveloperClient', () => {
+  const developerClient = new DeveloperClient(
+    new NEOONEDataProvider({
+      network: 'net',
+      rpcURL: 'rpc',
+    }),
+  );
+
+  test('startConsensusNow', () => {
+    // $FlowFixMe
+    developerClient.developerProvider.startConsensusNow = jest.fn();
```

Comment:
When you change `_developerProvider` above, remember don't depend on the internal implementation. Mock out `NEOONEDataProvider` separately from `developerClient`.